pkg/operator/sync.go

```diff
@@ -31,7 +31,10 @@ const (
 	machineExposeMetricsPort            = 8441
 	machineSetExposeMetricsPort         = 8442
 	machineHealthCheckExposeMetricsPort = 8444
-	defaultHealthPort                   = 9440
+	defaultMachineHealthPort            = 9440
+	defaultMachineSetHealthPort         = 9441
+	defaultMachineHealthCheckHealthPort = 9442
+	healthFailureThreshold              = 10
```

Comment:
where is this magic number 10 coming from?

Reply:
I saw that the default 3 retries was not enough, and the container failed couple of times before becoming ready. Just decided to give it more time. Not that important

Reply:
so why was it not enough? can we add a comment explaining why we choose 10?